base_report_to_printer/report.py

```diff
@@ -82,6 +83,15 @@ def get_pdf(self, cr, uid, ids, report_name, html=None,
         can_print_report = self._can_print_report(cr, uid, ids,
                                                   behaviour, printer, document,
                                                   context=context)
+        if report.usage == 'zebra' and printer:
+            arch = self.get_html(
+                cr, uid, ids, report_name, data=data, context=context)
+            tree = etree.fromstring(arch)
+            code = tree.xpath("//div[@class='code']")
+            res = code[0].text if code else ''
+            res = '\n'.join([l.strip() for l in res.split('\n') if l.strip()])
+            res += '\n'
+            return res
```

Comment:
I think this validation is no necessary because this variable is by default skipped
This is the [validation](https://github.com/Vauxoo/report-print-send/blob/8.0/base_report_to_printer/report.py#L62)

and this is always  [skipped](https://github.com/Vauxoo/report-print-send/blob/8.0/base_report_to_printer/report.py#L35) at least by context